charts/fluentd-elasticsearch/templates/prometheusrule.yaml

```diff
@@ -11,77 +11,9 @@ metadata:
     {{- end }}
   namespace: {{ .Values.prometheusRule.prometheusNamespace }}
 spec:
+  {{- with .Values.prometheusRule.rules }}
```

Comment:
thats superfluous. If we don't want the rules to be used we can just use .Values.prometheusRule.enabled=false